components/infobox/commons/infobox_patch.lua

```diff
@@ -43,11 +43,24 @@ function Patch:createInfobox()
 
 	if Namespace.isMain() then
 		infobox:categories('Patches')
+		self:addToLpdb(infobox, args)
 	end
 
 	return infobox:build()
 end
 
+--- Allows for overriding this functionality
+function Patch:addToLpdb(infobox, args)
+	local date = args.release
+	local monthAndDay = mw.getContentLanguage():formatDate('m-d', date)
```

Comment:
Why not ISO date?

Reply:
that is already saved into date
monthAndDay is for searching for certain dates independent of year

Reply:
Oh of course!